conda-store-server/conda_store_server/__init__.py

```diff
@@ -1,11 +1,12 @@
 import datetime
 import typing
-from pathlib import Path
+
+import platformdirs
 
 __version__ = "2024.3.1"
 
 
-CONDA_STORE_DIR = Path.home() / ".conda-store"
+CONDA_STORE_DIR = platformdirs.user_data_path(appname="conda-store")
```

Comment:
```suggestion
CONDA_STORE_DIR = Path(platformdirs.user_data_path(appname="conda-store"))
```

Just realized `platformdirs` returns `str` objects, so let's keep the same type around just in case.

Reply:
No change is necessary here, the `*_path` functions already return a `Path`.